pkg/events/events.go

```diff
@@ -176,7 +178,25 @@ func New(ctx context.Context, name, description string, opts ...Option) Event {
 	return (&definition{name: name, description: description}).New(ctx, opts...)
 }
 
+var errMarshalData = errors.Define("marshal_data", "failed to marshal data")
```

Comment:
An error implies failure, omit "failed to" etc in error messages
```suggestion
var errMarshalData = errors.Define("marshal_data", "marshal data")
```

Reply:
Done.